src/components/common/YourCompany/OfficeAddresses.vue

```diff
@@ -274,7 +274,7 @@
             <div class="meta-container">
               <label>Delivery Address</label>
               <div class="meta-container__inner">
-                <div class="form__row">
```

Comment:
This is one of our base classes. I renamed it so it doesn't look like one of Vuetify's classes (the ones that are rendered in the browser).